content/docs/reference/httpd-reference.md

```diff
@@ -23,11 +23,14 @@ version, see the [release
 notes](https://github.com/paketo-buildpacks/httpd/releases).
 
```

Comment:
Could you add a section with a table for the supported service binding? (e.g. https://paketo.io/docs/reference/dotnet-core-reference/#supported-service-bindings) Similarly, it could be nice to include a table that enumerates all the env vars that can be used to configure the buildpack with brief descriptions of what they're for and what their default values are. Something like the metadata included in the `metadata.configurations` section of the Java buildpacks' `bulidpack.toml` (e.g. [gradle](https://github.com/paketo-buildpacks/gradle/blob/7f17e4553536bc1c7e2e9bb5e64995f7ed1803e0/buildpack.toml#L34))

Reply:
I have added the service bindings table. In looking at the other buildpacks I see that they don't have the configuration environment variables listed. Is that a trend we would like to start?

Reply:
I think that's something we hoped for in the reference documentation. For instance, it's mentioned in the auto-reference docs [RFC](https://github.com/paketo-buildpacks/rfcs/blob/cb23c75ab7389bd11f7ca735e27a3caf9df81dc0/text/0040-auto-reference-docs.md). For language families that have a bunch of component buildpacks, this is hard to keep up to date manually, but I figure for something like standalong HTTPD it's not too painful to enumerate all the config options. Does that sound reasonable?

Reply:
Sure thing. I can try and add something like that.

Reply:
I have added something in the latest commit please feel free to check it out and give me any feedback.